storage/storage.go

```diff
@@ -255,6 +256,26 @@ func (storage DBStorage) ListOfClustersForOrg(orgID types.OrgID) ([]types.Cluste
 	return clusters, nil
 }
 
+// GetOrgIDByClusterID reads OrgID for specified cluster
+func (storage DBStorage) GetOrgIDByClusterID(cluster types.ClusterName) (types.OrgID, error) {
+	rows, err := storage.connection.Query("SELECT org_id FROM report WHERE cluster = $1 ORDER BY org_id LIMIT 1", cluster)
+	if err != nil {
+		return 0, err
+	}
+	defer closeRows(rows)
+
+	var orgID uint64
+	for rows.Next() {
```

Comment:
`if rows.Next()`

Reply:
For rows mostly used `for` to check more then one, but yes, here can be `if` as well